<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11099" started-at="2018-07-10T10:56:17Z" finished-at="2018-07-10T10:56:28Z">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:selenium.RunTest@7cd62f43]" name="feature" class="cucumber.api.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Default test" duration-ms="11099" started-at="2018-07-10T10:56:17Z" finished-at="2018-07-10T10:56:28Z">
      <class name="selenium.RunTest">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:selenium.RunTest@7cd62f43]" name="setUpClass" is-config="true" duration-ms="203" started-at="2018-07-10T16:26:17Z" finished-at="2018-07-10T16:26:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:selenium.RunTest@7cd62f43]" name="feature" duration-ms="10810" started-at="2018-07-10T16:26:17Z" description="Runs Cucumber Feature" data-provider="features" finished-at="2018-07-10T16:26:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Form Authentication]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[org.junit.ComparisonFailure: expected:<...://10.0.31.161:9292/[secure]> but was:<...://10.0.31.161:9292/[login]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: org.junit.ComparisonFailure: expected:<...://10.0.31.161:9292/[secure]> but was:<...://10.0.31.161:9292/[login]>
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.junit.ComparisonFailure: expected:<...://10.0.31.161:9292/[secure]> but was:<...://10.0.31.161:9292/[login]>
	at org.junit.Assert.assertEquals(Assert.java:125)
	at org.junit.Assert.assertEquals(Assert.java:147)
	at stepdefs.LoginPage.i_am_able_to_login_using_correct_credentials(LoginPage.java:65)
	at âœ½.Then I am able to access the secure area(src/test/java/resources/Module1.feature:14)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:selenium.RunTest@7cd62f43]" name="tearDownClass" is-config="true" duration-ms="14" started-at="2018-07-10T16:26:28Z" finished-at="2018-07-10T16:26:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- selenium.RunTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
